pkgs/by-name/ob/obj-magic/package.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation {
+  pname = "obj-magic";
+  version = "unstable-2020-09-20";
+
+  src = fetchFromGitHub {
+    owner = "tapio";
+    repo = "obj-magic";
+    rev = "f25c9b78cee6529a3295ed314d1c200677dc56c0";
+    hash = "sha256-4A8TasyLOh6oz21/AwBbE5s3055EPftFh8mymrveTvY=";
+  };
+
+  buildPhase = ''
+    runHook preBuild
+    ./make.sh
+    runHook postBuild
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    install -D obj-magic $out/bin/obj-magic
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "obj-magic is a command line tool for manipulating Wavefront OBJ 3d meshes";
```

Comment:
You should remove the package name from the beginning and change `3d` to `3D`

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes
 * `meta.description` must:
   * Be capitalized.
   * Not start with the package name.
     * More generally, it should not refer to the package name.